content/release-10-16-0/devices-10-16-0-bundle/10-16-0-0.md

```diff
@@ -205,6 +205,18 @@ DM-1541</td>
 <td>DM-1565</td>
 </tr>
 
+<tr>
+<td>Device Management</td>
+<td>Fixed an issue where users could not find a location on the map based on the address they entered.</td>
+<td>MTM-49644</td>
+</tr>
+
+<tr>
+<td>Device Management</td>
+<td>In case of many active alarms of different severities, in the "Active critical alarms" widget users might have needed to press the <b>Load more</b> button several times on an empty list in order to see their active critical alarms. This is now fixed - the widget always initially shows active critical alarms if available and the<b>Load more</b> button does not load empty pages.</td>
```

Comment:
```suggestion
<td>In case of many active alarms of different severities, in the "Active critical alarms" widget users might have needed to press the <b>Load more</b> button several times on an empty list in order to see their active critical alarms. This is now fixed - the widget always initially shows active critical alarms if available and the <b>Load more</b> button does not load empty pages.</td>
```